### PR TITLE
Adding Initial Testing Structure

### LINKED ISSUE #6: Add Tests
The purpose of each codemod is pretty opaque, so we can both test the modification and get some built-in examples if we're able to build out a test suite.

### PR DESCRIPTION
**Pull request by joshblack**
Copied the structure from [`react-codemod`](https://github.com/reactjs/react-codemod) and applied it to the `rackt-codemod` repo. These changes bring in `jest` into the project, and an `npm test` script that was copied over from `react-codemod` along with test files for each transform in the project.

Closes #6.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "rimraf transforms && babel modules/ -d transforms",
     "lint": "eslint modules",
     "prepublish": "npm run build",
-    "test": "npm run lint"
+    "test": "f() { EXIT=0; npm run lint || EXIT=$?; NODE_ENV=test jest $@ || EXIT=$?; exit $EXIT; }; f"
```
Comment (taion) [inline package.json:12]: Are we okay with tests not being runnable on Windows?

Comment (joshblack) [inline package.json:12]: Can switch over to just npm scripts if you want!

**Comment by taion** [comment]
Haha actually forget my dumb little nit. This is really great, thanks so much for PRing this!

**Comment by joshblack** [comment]
:tada: :tada: :tada: anytime!

**Pull request merged**